.github/pull_request_template.md

```diff
@@ -5,26 +5,28 @@
 ## Description
 <!-- Workshop description that you'll use in advertisements -->
 
+
+# Tasks
+More detailed instructions for specific steps here: <https://cct-datascience.github.io/group-procedures/workshops.html>
+
 ## Pre-workshop checklist:
 
-- [ ] Create a Zoom meeting with registration
+- [ ] Create a Zoom meeting with registration ([instructions & settings](https://cct-datascience.github.io/group-procedures/workshops.html#creating-a-template-for-workshop-zoom-meetings))
 - [ ] Write up announcement and record on the [Workshops HackMD](https://hackmd.io/-hSaZbuzQI-gAPP3pASvRw)
 - [ ] Advertise
```

Comment:
For emails and other announcements (such as on Slack, Mastodon, etc), what's the timing of the announcements? For example:
- within one week after workshop date finalization, send announcement
- 3 weeks before workshop, send reminder
- etc